### PR TITLE
chore - create auth gha

### LINKED ISSUE #1: Create Prefect Auth GHA
Create a GHA that runs the Prefect Auth commands to sign into prefect cloud

### PR DESCRIPTION
**Pull request by jimid27**
- Resolves: https://github.com/PrefectHQ/actions-prefect-auth/issues/1

### CODE REVIEW

File: .github/workflows/add-to-platform-project.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    permissions:
+      issues: read
+      repository-projects: write
+
+    steps:
+      - uses: actions/add-to-project@v0.5.0
+        with:
+          project-url: https://github.com/orgs/PrefectHQ/projects/44
+          github-token: ${{ secrets.GHA_ADD_TO_PROJECT_PROJECTS_RO_REPOS_RW }}
```
Comment (jimid27) [inline .github/workflows/add-to-platform-project.yaml:25]: this is an org level secret, right?

Comment (jamiezieziula) [reply .github/workflows/add-to-platform-project.yaml:25]: ya

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```
Comment (jamiezieziula) [inline .github/dependabot.yaml]:
```suggestion
  - package-ecosystem: github-actions
    directory: /
    schedule:
      interval: monthly
```

File: .github/workflows/yaml-lint.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: lint yaml files
+
+"on":
+  pull_request:
+    branches: [main]
```
Comment (jamiezieziula) [inline .github/workflows/yaml-lint.yaml]:
```suggestion
    branches:
      - main
```

**Review by parkedwards — APPROVED** [review]

**Comment by parkedwards** [comment]
clean

**Review by jamiezieziula — APPROVED** [review]

**Pull request merged**